Assignment2/solrac1999-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,47 @@
+{
+  "@context": {
+  "prefix": "http://example.org/components/",
+  "xsd": "http://www.w3.org/2001/XMLSchema#"
+  },
+  "@graph0": [
+    {
+      "@id": "prefix:Class01",
+      "prefix:includes": [
+        {
+          "@id": "prefix:Sensor029"
+        },
+        {
+          "@id": "prefix:Computer101"
+        }
+      ]
+      "@id": "prefix:Sensor029",
+      "prefix:hasMeasurement":
+      {
+        "@id": "prefix:Measurement8401"
+      }
+      "@id": "prefix:Computer101",
+      "prefix:hasOwner":
+      {
+        "@id": "prefix:User10A"
+      }
+      "@id": "prefix:Measurement8401",
+      "prefix:hasTemperature":
+      {
+        "@type: ": "xsd:integer",
+        "@value": "29"
+      }
+      "@id": "prefix:Measurement8401",
+      "prefix:atTime":
+      {
+        "@type: ": "xsd:dataTime",
+        "@value": "2010-06-12T12:00:12"
+      }
+      "@id": "prefix:User10A",
+      "prefix:hasName":
+      {
+        "@type: ": "xsd:string",
+        "@value": "Pedro"
+      }
+    },
+  ]
+}

```

Comment:
Validate the file because you have other syntactical errors. You might use json-ld playground.